sale_loyalty_auto_refresh/tests/test_sale_loyalty_auto_refresh.py

```diff
@@ -0,0 +1,142 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import new_test_user, tagged
+
+from odoo.addons.sale_loyalty.tests.common import TestSaleCouponCommon
```

Comment:
It'd be nice to have a diff here. Aside from that, at first glance I see lot of changes:

- Copyright and licences aren't respected from the original
- Tests cases are trimmed so you're not testing the same effects as before, so avoiding regression isn't granted or hard to tell...

Reply:
Sorry about the Copyright part, i may remove it by mistake.
I just add 1 more unit test to check the case of having more than 1 applicable promotion.